miner/scroll_worker.go

```diff
@@ -558,10 +558,13 @@ func (w *worker) newWork(now time.Time, parent *types.Block, reorging bool, reor
 		// clique with relaxed period uses time.Now() as the header.Time, calculate the deadline
 		deadline = time.Unix(int64(header.Time+w.chainConfig.Clique.Period), 0)
 	}
-	if w.chainConfig.SystemContract != nil && w.chainConfig.SystemContract.RelaxedPeriod {
+	if w.chainConfig.SystemContract != nil && w.chainConfig.SystemContract.RelaxedPeriod && w.chainConfig.SystemContract.Period != 1 {
```

Comment:
Why do we need to add `&& w.chainConfig.SystemContract.Period != 1` here?

Reply:
update [71dd65e](https://github.com/scroll-tech/go-ethereum/pull/1174/commits/71dd65e739c60304ddb2179dde30d4a8b893ef12)